docker-compose.yml

```diff
@@ -1,14 +1,24 @@
 version: '3'
+
 services:
-  db:
-    restart: always
+  app:
+    build: .
+    env_file:
+      - .env
+    volumes:
+      - .:/src
```

Comment:
this doesn't currently make the code reload live because gunicorn isn't watching for changes. It would be nice if we did have live-reloading for development though, should we switch to using the django dev server when running via docker compose?

Reply:
oh, crap, you're totally right! it should be gunicorn in prod and runserver for local dev. I can adjust the server script.